README.md

```diff
@@ -1,13 +1,24 @@
 # Edge, AI & Security Demo
 
-This repository contains the files needed to create a demonstration of an
-OpenShift Cluster running an AI / Edge workload, protected by Keylime.
+This repository contains the files and documentation to perform the joint Edge /
+AI, Security demo developed within OCTO, Emerging Technologies.
 
-To create the container images, use the following command line syntax:
+The demo consists of a Edge / AI based application. The application streams
```

Comment:
NIT: Should be "an Edge / AI"

Reply:
Thanks Michael, made the fix, could you approve again pls?